work_queue/src/work_queue.h

```diff
@@ -244,9 +244,8 @@ struct work_queue_stats {
 	int capacity_cores;     /**< The estimated number of workers' cores that this master can effectively support.*/
 	int capacity_memory;    /**< The estimated number of workers' MB of RAM that this master can effectively support.*/
 	int capacity_disk;      /**< The estimated number of workers' MB of disk that this master can effectively support.*/
-	int capacity_instantaneous; /**< The estimated number of tasks that this master can support considering only the most recently completed task. */
```

Comment:
Changes in names of fields, or order of fields should be avoided in wq_stats.

Reply:
I removed capacity_instantaneous since it is never used. We had anticipated we would use this with the capacity code, but that never came to fruition. I am alright with keeping it in just in case we would like to use it later. Thoughts?

Reply:
capacity_instantaneous is still kept currently, but it is renamed.

Reply:
Don't rename it. We don't want abbreviated names in the stats.